packages/vulcan-backoffice/lib/components/BackofficeLayout.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect, useRef } from 'react';
+import {
+  getAuthorizedMenuItems,
+  menuItemProps,
+  registerComponent,
+  withCurrentUser,
+  Components,
+} from 'meteor/vulcan:core';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
```

Comment:
No you can't do that. In core packages you don't have access to a styling solution, you should expect Bootstrap/Material UI to style the component. You can use the `registerComponent` pattern so that your component can be overriden by those packages in order to add style.
You shouldn't have any dependency to styled-components in the code.